script/src/verify.rs

```diff
@@ -272,7 +273,7 @@ impl<'a, DL: DataLoader> TransactionScriptsVerifier<'a, DL> {
         // Now run each script group
         for group in self.lock_groups.values().chain(self.type_groups.values()) {
             let result = if group.script.code_hash() == TYPE_ID_CODE_HASH.pack()
-                && group.script.hash_type().unpack() == ScriptHashType::Type
+                && group.script.hash_type() == Into::<u8>::into(ScriptHashType::Type)
```

Comment:
what about `ScriptHashType::Data.into()`?

Reply:
rust compiler is not smart enough for this operation, have to provide a specific type here.